include/tvm/meta_schedule/database.h

```diff
@@ -357,6 +357,18 @@ class Database : public runtime::ObjectRef {
    */
   TVM_DLL static Database JSONDatabase(String path_workload, String path_tuning_record,
                                        bool allow_missing);
+  /*!
+   * \brief Create a database merged from multiple databases.
+   * \param preferred The preferred database. If the preferred database responses to a query,
+   * all other databases will be ignored.
+   * \param databases The databases to be merged.
+   * \param fallback The fallback databases. If all the databases didn't answer a query,
+   * the fallback databases will be used.
+   * \return The merged database.
+   */
+  TVM_DLL static Database MergedDatabase(Array<Database, void> preferred,
+                                         Array<Database, void> databases,
+                                         Array<Database, void> fallback);
```

Comment:
Just curious, what is the second template parameter `void` used for?

Reply:
`Database` at that moment is an incomplete type, which won't pass Array's static type checking which requires the `Database` to be a subclass of `ObjectRef`, and therefore, we use the void-trick here